pywren/pywren_ibm_cloud/storage/cos_backend.py

```diff
@@ -34,32 +34,13 @@ class COSBackend(object):
 
     def __init__(self, cos_config):
 
-        if 'cos_endpoint' in cos_config:
-            service_endpoint = cos_config.get('cos_endpoint').replace('http:', 'https:')
+        service_endpoint = cos_config.get('endpoint').replace('http:', 'https:')           
 
-        elif {'cos_endpoints', 'cos_region'} <= set(cos_config):
-            endpoints = requests.get(cos_config.get('cos_endpoints')).json()
-            region = cos_config.get('cos_region')
-
-            if region in endpoints['service-endpoints']['cross-region']['us']['public']:
-                cos_host = endpoints['service-endpoints']['cross-region']['us']['public'][region]
-
-            elif region in endpoints['service-endpoints']['cross-region']['eu']['public']:
-                cos_host = endpoints['service-endpoints']['cross-region']['eu']['public'][region]
-            
-            elif region in endpoints['service-endpoints']['regional']:
-                cos_host = endpoints['service-endpoints']['regional'][region]['public'][region]
-
-            elif region in endpoints['service-endpoints']['regional']:
-                cos_host = endpoints['service-endpoints']['regional'][region]['public'][region]
-
-            service_endpoint = 'https://' + cos_host
-
-        if 'cos_api_key' in cos_config:
```

Comment:
@JosepSampe why we removed public?

Reply:
This configuration is never used. I write it in a very preliminary version, but it is not necessary as we never use **region** in config file.

Reply:
@JosepSampe 👍